cmake/Modules/FindJson.cmake

```diff
@@ -0,0 +1,15 @@
+# - Try to find nlohmann_json
+# Once done, this will define
+#
+#  Json_FOUND - system has nlohmann_json
+#  Json_INCLUDE_DIRS - the nlohmann_json include directories
+#  Json_LIBRARIES - link these to use nlohmann_json
+#
+# See documentation on how to write CMake scripts at
+# http://www.cmake.org/Wiki/CMake:How_To_Find_Libraries
+
+include(LibFindMacros)
+
+libfind_pkg_detect(Json nlohmann_json FIND_PATH nlohmann/json.hpp)
+set(Json_VERSION ${Json_PKGCONF_VERSION})
+libfind_process(Json)
```

Comment:
I don't really get the point of this file. why not simply use `find_package(nlohmann_json 3.10.5 REQUIRED)`

Reply:
This is why we use our own modules instead of a 'simple' find_package call indeed :)

Reply:
ok